conf/config.neon

```diff
@@ -1465,6 +1466,23 @@ services:
 		tags:
 			- phpstan.dynamicFunctionThrowTypeExtension
 
+	-
+		class: PHPStan\Type\Php\PregMatchTypeSpecifyingExtension
+		arguments:
+			disabled: %featureToggles.narrowPregMatches%
```

Comment:
This extension does not need "disabled" parameter. Check out the "conditionalTags" sections.

Reply:
I tried that with conditionaltags, but it did not work for `phpstan.functionParameterOutTypeExtension`.

Reply:
```
	PHPStan\Type\Php\PregMatchParameterOutTypeExtension:
		phpstan.functionParameterOutTypeExtension: %featureToggles.narrowPregMatches%
```

leads to

```
In Compiler.php line 293:
                                                                                                                                        
  Unexpected item 'conditionalTags › PHPStan\Type\Php\PregMatchParameterOutTypeExtension › phpstan.functionParameterOutTypeExtension'.  
                                                                                                                                        
```

Reply:
Whenever we add new tags, they need to be added to the ConditionalTagsExtension schema as well.

Reply:
thanks, thats it.